test/ui/ResultsFiltersPreferencesTest.ts

```diff
@@ -95,6 +95,22 @@ export function ResultsFiltersPreferencesTest() {
         let form = $$(test.cmp.element).find('form');
         expect(() => $$(form).trigger('submit')).not.toThrow();
       });
+
+      it('should not trigger a query when a full breadcrumb clear is triggered', () => {
+        test.env.queryController.firstQuery = true;
```

Comment:
Is it important to set `firstQuery` to `true` here?

Reply:
Yes, otherwise the test might be skewed when we create the dom elements, and the filter selection is applied.

By setting it as "firstQuery", it blocks the query when the element is first created.